docs: fix typos (#40342)
vercel/next.js: docs/testing.md

### Quickstart

> The fastest way to get started is to use `create-next-app` with the [with-playwright example](https://github.com/vercel/next.js/tree/canary/examples/with-playwright). This will create a Next.js project complete with Playwright all setup.

Reviewer: ```suggestion
The fastest way to get started is to use `create-next-app` with the [with-playwright example](https://github.com/vercel/next.js/tree/canary/examples/with-playwright). This will create a Next.js project complete with Playwright all set up.
```

  testEnvironment: 'jest-environment-jsdom',
}

> // createJestConfig is exported this way to ensure that next/jest can load the Next.js config which is an async

Reviewer: ```suggestion
// createJestConfig is exported this way to ensure that next/jest can load the Next.js config which is async
```

For more information on handling static assets, please refer to the [Jest Docs](https://jestjs.io/docs/webpack#handling-static-assets).

> **Optional: Extend Jest with custom matches**

Reviewer: ```suggestion
**Optional: Extend Jest with custom matchers**
```

See https://jestjs.io/docs/expect#:~:text=a%20number%20of%20%22-,matchers,-%22%20that%20let%20you

> `@testing-library/jest-dom` includes a set of convenient [custom matches](https://github.com/testing-library/jest-dom#custom-matchers) such as `.toBeInTheDocument()` making it easier to write tests. You can import the custom matches for every test by adding the following option to the Jest configuration file:

Reviewer: ```suggestion
`@testing-library/jest-dom` includes a set of convenient [custom matchers](https://github.com/testing-library/jest-dom#custom-matchers) such as `.toBeInTheDocument()` making it easier to write tests. You can import the custom matchers for every test by adding the following option to the Jest configuration file:
```
